### DISCUSSION TITLE
Github desktop VCRUNTIME140_1.dll Please help!

### DISCUSSION THREAD

**Discussion by Iplaygames995** [community/community#22065] [New to GitHub]
<p>So I am attempting to connect Projects from Unreal engine, using Github Desktop. I am able to see all his assets and everything, but when I try to open them I am greeted with this message.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/6/6a326f2ecc35875662e7947c804817cb14bd89c3.jpeg" data-download-href="/uploads/short-url/f9sFlsGVY2X3KNA7G9wU03dfnTt.jpeg?dl=1" title="VCRUNTIME140_1.dll fix" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/31864387/181068759-c449cc8b-97f8-4f28-9157-4a66095c48b9.jpeg" alt="VCRUNTIME140_1.dll fix" data-base62-sha1="f9sFlsGVY2X3KNA7G9wU03dfnTt" width="690" height="287" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/6/6a326f2ecc35875662e7947c804817cb14bd89c3_2_690x287.jpeg, https://aws1.discourse-cdn.com/github/optimized/2X/6/6a326f2ecc35875662e7947c804817cb14bd89c3_2_1035x430.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/6/6a326f2ecc35875662e7947c804817cb14bd89c3_2_1380x574.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/6/6a326f2ecc35875662e7947c804817cb14bd89c3_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">VCRUNTIME140_1.dll fix</span><span class="informations">3839×1600 2.65 MB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I am a complete beginner when it comes to this, so any help would be amazing!</p>
<p>P.S. I know that it says, "uninstalling may mix the program, but I’m wondering if there is any other possible ways, as reinstalling the program would erase a fair amount of work.</p>

**Comment by steveward** [comment, marked as answer]
<p><a class="mention" href="/iplaygames995">@Iplaygames995</a> check out <a href="https://answers.unrealengine.com/questions/796216/vcruntime140dll-is-missing-error-1.html">this forum page for Unreal Engine</a> to see how to resolve this. If you continue to have issues I would recommend reaching out to Unreal directly. Let me know if you have any specific questions about using GitHub Desktop.</p>
